src/lib/structures/Command.ts

```diff
@@ -499,6 +502,8 @@ export class Command<PreParseReturn = Args, O extends Command.Options = Command.
 			} else if (guildNewsThread) {
 				preconditions.append(CommandPreConditions.GuildNewsThreadOnly);
 			}
+
+			if (guildVoice) preconditions.append(CommandPreConditions.GuildVoiceOnly);
```

Comment:
```suggestion
			if (guildVoice) {
				preconditions.append(CommandPreConditions.GuildVoiceOnly);
			}
```

Reply:
Just out of curiosity, what is the difference or what benefits does it give?

Reply:
Readability